src/components/header/header.css

```
@@ -0,0 +1,3 @@
+h1 {
+  background-color: blue;
+}
```

Usa uma classe, seletor de tag costuma dar problema de conflito
```suggestion
h1.title {
  background-color: blue;
}
```